src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -26,5 +27,147 @@ public partial class AppDomain
                 AppContext.UnhandledException -= value;  
             }  
         }  
+
+        public string DynamicDirectory => null;
+        public string FriendlyName => "DefaultDomain";
+        public int Id => DefaultADID;
+        public bool IsFullyTrusted => true;
+        public bool IsHomogenous => true;
+        public bool ShadowCopyFiles => false;
+        public event System.EventHandler DomainUnload
+        {
+            add  
+            {  
+                throw new PlatformNotSupportedException();
+            }  
+  
+            remove  
+            {  
+                throw new PlatformNotSupportedException();
+            }  
+        }
+
+        public event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException
+        {  
+            add  
+            {  
+                AppDomainAugments.FirstChanceException += value;  
+            }  
+  
+            remove  
+            {  
+                AppDomainAugments.FirstChanceException -= value;  
```

Comment:
FirstChanceException and ProcessExit should be on AppContext (because of the unhandled exception is there already).